x/staking/client/cli/query.go

```diff
@@ -343,8 +343,11 @@ $ %s query staking unbonding-delegation cosmos1gghjut3ccd8ay0zduzj64hwre2fxs9ld7
 				return err
 			}
 
-			return cliCtx.PrintOutput(types.MustUnmarshalUBD(cdc, res))
-
+			if res != nil {
+				return cliCtx.PrintOutput(types.MustUnmarshalUBD(cdc, res))
+			} else {
+				return cliCtx.PrintOutput(nil)
```

Comment:
Would rather avoid using nil here.

Reply:
`cliCtx.PrintOutput` required a parameter which implement the interface `fmt.Stringer`. I can't find the right class to hold `"null"`.

Reply:
For example,

I tried querying for a non-existent delegation and I got the following:

```
$ gaiacli q staking delegation <del-addr> <val-addr>
ERROR: {"codespace":"staking","code":102,"message":"no delegation for this (address, validator) pair"}
```

We should have something identical for this command. Maybe look into how that's done :)

Reply:
Got it !

Reply:
Yeah so maybe we just need to return an error?

Reply:
I need to change `cliCtx.QueryStore()` to `cliCtx.QueryWithData()`. It will take sometime.